package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@honehealth/jutsu",
-  "version": "2.0.11-rc.1",
+  "version": "2.0.12-rc.1",
```

Comment:
Oh dear what happened to the semantic versioning of this? this should be 2.0.0-rc.12

Reply:
oops! I saw that there was a separate rc.1 counting all the way up to 11 so I did the same